std/check.asm

```diff
@@ -0,0 +1,9 @@
+/// This is a built-in function taking a string argument and terminating
+/// evaluation unsuccessfully with this argument as explanation.
+/// This symbol is not an empty array, the actual semantics are overridden.
+let panic = [];
```

Comment:
This is quite confusing

Reply:
The main other way I could think of is something like:
```
std::core:
let builtin = |name| (); // The only weird thing
std::check:
let panic = std::core::builtin("panic");
```

And then `builtin` would be the only real builtin function.

Reply:
although this would not pass strict type checking

Reply:
ok, we could use something like `let panic = std::core::builtin::<"panic">();`

Reply:
if it's already built-in, why not make it fully built-in?

Reply:
so we keep it like this for now?